### PR TITLE
Component loader

### PR DESCRIPTION
**Pull request by NancyRike**

### CODE REVIEW

File: src/pages/landingPage/LandingPage.js [hunk 1]
```diff
@@ -1,140 +1,175 @@
...
+};
 
-const mapStateToProps = (state)=>{
-    return{
-        popularMenu: state.meals.popularMenu
-    }
-}
-const connector = connect(mapStateToProps, {popularMenuAction})
+const mapStateToProps = (state) => {
+  console.log(state.loader.loading, "ughhhh");
```
Comment (Dverybest) [inline src/pages/landingPage/LandingPage.js]: Remove console.log

File: src/pages/landingPage/LandingPage.js [hunk 2]
```diff
@@ -1,140 +1,175 @@
...
-                        data.map((item, index) => {
-                            return (
-                                <Card key={index} {...item} />
-                            )
-                        })
-                    }
-                </div>
-            </section>
+              ) : (
+                props.popularMenu?.map((dish, index) => {
```
Comment (Dverybest) [inline src/pages/landingPage/LandingPage.js]:
```suggestion
                props.popularMenu?.slice(6).map((dish, index) => {
```
can we slice this popularMenu array to 6 as suggest above

**Review by Dverybest — COMMENTED** [review]
Looks good to me, just a little change and screenshot/screen recording required.

File: src/pages/landingPage/LandingPage.js (hunk 2 above)
Comment (NancyRike) [reply src/pages/landingPage/LandingPage.js]: is there any reason why we would want that?

Comment (Dverybest) [reply src/pages/landingPage/LandingPage.js]: Yes... the landing page should not have plenty things in it... People should go to menu to view plenty menu

**Review by Dverybest — APPROVED** [review]

**Pull request merged**